setup.py

```diff
@@ -40,14 +40,4 @@
     },
     include_package_data=True,
     license='GPL-2.0-only',
-    install_requires=[
-        'pyyaml'
-    ],
-    # test_suite='tests',
-    # tests_require=[
-    #     'coverage',
-    #     'pytest-cov',
-    #     'pytest',
-    #     'tox',
-    # ]
```

Comment:
This breaks pipx and a various of tools which invoke setuptools.

Reply:
ok, check now